index.html

```diff
@@ -53,10 +53,87 @@
       {
         color:red;
       }
+
+
     </style>
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
     <script>
-      // Put some code in here!
+
+    $(document).ready(function(){
+      var counter = $("#counter")
+      var startNum = 1234;
+      counter.val(startNum)
+      var gameInProgress = false;
+      var countID;
+
+    function countDown(){
```

Comment:
Watch your indentation please! In Atom if you go to Edit > Lines > Autoindent it'll tidy everything up for you.